spec/content_metadata_spec.rb

```diff
@@ -726,7 +726,6 @@
 
     context 'when not all input files exist' do
       it 'does not generate valid content metadata' do
-        expect(File.exist?(TEST_TIF_INPUT_FILE)).to be true
```

Comment:
this is silly - why test that our own test data files exist?   If they don't, we'll hear about it from the spec failing.